UI/src/components/Form.jsx

```diff
@@ -0,0 +1,265 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {
+    // faTwitter,
```

Comment:
Should we just remove all the imported icons here, rather than commenting them out?